<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sql</class>
 <widget class="QWidget" name="Sql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Execution of any SQL-query</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter a SQL - query</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtSql"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnExecute">
       <property name="text">
        <string>Execute</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/images/images/applications.png</normaloff>:/images/images/applications.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QTableView" name="tblData">
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>20</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
